CFC_DataCollector/recommender/recommendation_pipeline.py

```diff
@@ -14,26 +15,24 @@ def retrieve_alternative_trips(self, trip_list):
         return []
 
     def get_selected_user_utility_model(self, user_id):
-        return []
+        return UserUtilityModel.find_from_db(user_id)
 
     def recommend_trips(self, trip_id, utility_model):
         return []
```

Comment:
Wait, that's not the issue. At query time, trips and alternatives are
linked. The issue is when we throw it all into the model building phase

On Wednesday, April 29, 2015, shankari <notifications@github.com> wrote:

> In CFC_DataCollector/recommender/recommendation_pipeline.py
> <https://github.com/e-mission/e-mission-server/pull/111#discussion_r29347047>
> :
>
> > @@ -14,26 +15,24 @@ def retrieve_alternative_trips(self, trip_list):
> >          return []
> >
> >      def get_selected_user_utility_model(self, user_id):
> > -        return []
> > +        return UserUtilityModel.find_from_db(user_id)
> >
> >      def recommend_trips(self, trip_id, utility_model):
> >          return []
>
> Sorry, I am not sure I remember the discussion.
> How about something naive like adding an "original_trip" field to each
> alternative?
> It should be pretty easy to then query the alternatives db for all
> alternatives for an original trip, and to lookup the original trip from the
> alternatives.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/111/files#r29347047>.
>